### PR TITLE
Send client access logs too in stackdriver

### PR DESCRIPTION
**Pull request by gargnupur**
Signed-off-by: gargnupur <gargnupur@google.com>

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by bianpengyuan — COMMENTED** [review]
Can we add deprecation warning in the same pr?

File: operator/pkg/apis/istio/v1alpha1/values_types.proto [hunk 1]
```diff
@@ -808,11 +808,23 @@ message TelemetryV2PrometheusConfig {
   ConfigOverride config_override = 3;
 }
 
+// Types of Access logs to export.
+  enum AccessLogging {
```
Comment (bianpengyuan) [inline operator/pkg/apis/istio/v1alpha1/values_types.proto:812]: Should this enum be put into TelemetryV2StackDriverConfig? Seems to be sd only enum.

Comment (gargnupur) [reply operator/pkg/apis/istio/v1alpha1/values_types.proto:814]: done

**Review by bianpengyuan — APPROVED** [review]
looks good, although the make gen is still failing.

**Review by richardwxn — APPROVED** [review]

**Pull request merged**